### PR TITLE
feat: Added tidyverse page under `software_development/languages/R`

### PR DESCRIPTION
**Pull request by mtran-code**
- Added page on what the tidyverse is

### CODE REVIEW

**Comment by mtran-code** [comment]
I'm thinking I might've not followed KISS, might cut it down a bit.

File: docs/software_development/Languages/R/Tidyverse/index.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+- **Compatibility**: The Tidyverse is not the only way to do things in R, and as
+    well, there are countless other packages that may not play nicely with
+    Tidyverse-oriented data structures or tibbles. In these cases you may have
+    to convert back and forth between tibbles and data frames, which can be
+    cumbersome.
+
+## Introductory Example
+
+Let's say we have a CSV file containing a table, and we want to read into R,
+perform some manipulations on the data, and then write it out to a new file.
```
Comment (jjjermiah) [inline docs/software_development/Languages/R/Tidyverse/index.md:91]:
```suggestion
Let's say we have a CSV file containing a table, and we want to:

1.  read into R;
2.  perform some manipulations on the data; and then
3.  write it out to a new file.
```

Comment (jjjermiah) [reply docs/software_development/Languages/R/Tidyverse/index.md:91]: suggestion for easily understanding task

**Review by jjjermiah — APPROVED** [review]

**Comment by jjjermiah** [comment]
lmk if the others are planning on reviewing before I merge

**Comment by mtran-code** [comment]
> lmk if the others are planning on reviewing before I merge

Not that I know of, up to you if you think your review is fine

**Pull request merged**